Mathlib/RingTheory/WittVector/Isocrystal.lean

```diff
@@ -76,11 +77,13 @@ variable [IsDomain k] [CharP k p] [PerfectRing k p]
 def FractionRing.frobenius : K(p, k) ≃+* K(p, k) :=
   IsFractionRing.ringEquivOfRingEquiv (frobeniusEquiv p k)
 
-/-- The Frobenius automorphism of `k` induces an endomorphism of `K`. For notation purposes. -/
+/-- The Frobenius automorphism of `k` induces an endomorphism of `K`. For notation purposes.
+Notation `φ(p, k)` in the Isocrystal namespace. -/
 def FractionRing.frobeniusRingHom : K(p, k) →+* K(p, k) :=
   FractionRing.frobenius p k
 
-scoped[Isocrystal] notation "φ(" p ", " k ")" => WittVector.FractionRing.frobeniusRingHom p k
+@[inherit_doc] scoped[Isocrystal]
```

Comment:
Usually `scoped` is on the same line as `notation`. What is the reason for this choice?